src/plugins/console/public/application/components/network_request_status_bar/network_request_status_bar.tsx

```diff
@@ -80,7 +80,10 @@ export const NetworkRequestStatusBar: FunctionComponent<Props> = ({
               }`}</EuiText>
             }
           >
-            <EuiBadge color={mapStatusCodeToBadgeColor(statusCode)}>
+            <EuiBadge
+              color={mapStatusCodeToBadgeColor(statusCode)}
+              data-test-subj="consoleRequestStatusBadge"
```

Comment:
I could have sworn I saw you implement this already in another PR. Or am I imagining things? 😅

Reply:
Oh! It was in a parallel PR that just got merged.